content/publication-bias.md

```diff
@@ -0,0 +1,27 @@
+# Publication bias
+
+Publication bias is the selective publication of research results on an arbitrary judgement. Such arbitrary judgements (can) vary across individuals, over time, and remain undisclosed to those who are not privy to the decision process. Publication bias distorts the complete picture of research results and by extension threatens the validity of results, theories, and effectivity of interventions. For example, [less than half of all registered clinical trials on breast cancer are published](https://doi.org/10.1186/s41073-016-0017-4) for unknown reasons.
+
+The most well-known form of publication bias is based on statistical significance: Significant results (p < .05) have a higher probability of getting submitted- and accepted for publication than nonsignificant results. Other forms of publication bias can include higher probabilities of publication for results that confirm prior beliefs or research that uses common methods because they're better understood. 
+
+Selective publication can be with intent, but can also be unconscious. Intentional selectivity can occur when there is a [Conflict of interest](conflict-of-interest.md). For example, a policy research institute might be commissioned to do research for a company that pressures them to show specific results only. Unconscious selectivity can occur due to implicit assumptions in the quality of a study. For example, publication bias based on statistical significance is often the result of an implicit assumption that study quality is somehow based on the results that study produces.
```

Comment:
Maybe include example of funding bias? (eg tobacco industry / http://www.cochrane.org/MR000033/METHOD_industry-sponsorship-and-research-outcome)

Reply:
Saving this for Conflict of Interest #41